src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestCase.java

```diff
@@ -1338,12 +1338,12 @@ public void test_fetch_with_prune() throws Exception {
         /* master -> branch1 */
         /*        -> branch2 */
         w.init();
+        w.git.setRemoteUrl("origin", bare.repoPath());
         w.touch("file-master", "file master content " + java.util.UUID.randomUUID().toString());
         w.git.add("file-master");
         w.git.commit("master-commit");
         ObjectId master = w.head();
-        assertThat(getBranchNames(w.git.getBranches()), contains("master"));
-        w.git.setRemoteUrl("origin", bare.repoPath());
+        assertEquals("Wrong branch count", 1, w.git.getBranches().size());
```

Comment:
The change in this line is unrelated to the actual change described in the commit message. It reverts a former change.